mne/io/tests/test_reference.py

```diff
@@ -238,8 +238,12 @@ def test_set_eeg_reference():
         set_eeg_reference(raw, ['EEG 001'], True, True)
 
 
-@pytest.mark.parametrize('ch_type', ('auto', 'ecog', 'dbs'))
-def test_set_eeg_reference_ch_type(ch_type):
+@pytest.mark.parametrize('ch_type, msg',
+    [('auto', ('ECoG',)),
+     ('ecog', ('ECoG',)),
+     ('dbs', ('DBS',)),
+     (['ecog', 'dbs'], ('ECoG', 'DBS'))])
+def test_set_eeg_reference_ch_type(ch_type, msg):
```

Comment:
oops, I broke style rules. This should fix it?

```suggestion
@pytest.mark.parametrize('ch_type, msg',
                         [('auto', ('ECoG',)),
                          ('ecog', ('ECoG',)),
                          ('dbs', ('DBS',)),
                          (['ecog', 'dbs'], ('ECoG', 'DBS'))])
def test_set_eeg_reference_ch_type(ch_type, msg):
```